config/locales/modules/personal-social-and-emotional-development.yml

```diff
@@ -809,103 +809,157 @@ en:
+          Children in this category did not learn that the adult’s role was to care for them. This could have been as a result of trauma or emotionless care. A significant role model may have unintentionally taught the child that it is not good to be emotional. The caring style will have been very practical. The child will develop self-reliance and conclude that other people are not able to help them.
 
           Children who have experienced this early attachment style will most likely:
 
-          - be anxious to please
-          - be anxious that others are not able to keep them safe
-          - feel rejected
-          - struggle to connect with others
-
-          ##Insecure ambivalent attachment
+            - be anxious to please
+            - be anxious that others are not able to keep them safe
+            - feel rejected
+            - struggle to connect with others
+      3-2-1-3e:
+        heading: Insecure ambivalent attachment
+        body: |
           ![An anxious-looking young child, holding on to an adult's hand but looking behind them.](/assets/3-2-1-3a-1305634049.jpg)
 
-          This type of attachment is also known as anxious attachment. Children develop this attachment style when they cannot predict the response they will receive from an adult. This may be as a result of inconsistent care, resulting in unpredictability and a lack of trust. The significant adult role in the child’s life may have easily swung between kindness and reward and punishment, with no sense of reason for the child.
+          During the strange situation experiment, some babies were anxious about the experiment and wary of the stranger. When their mother was there, they tended to stick close to her for comfort and security instead of exploring the room.
+
+          When their mother left, these babies were very upset. However, when she returned they had mixed reactions. Some rushed over to their mother, but could not be successfully comforted by her. Some seemed angry at their mother for leaving and avoided being comforted by her.
+
+          Ainsworth classified these babies as having an insecure ambivalent attachment style. This type of attachment is also known as anxious attachment.
+
+          Children develop this attachment style when they cannot predict the response they will receive from an adult. This may be as a result of inconsistent care, resulting in unpredictability and a lack of trust. The significant adult role in the child's life may have easily swung between kindness and reward and punishment, with no sense of reason for the child.
 
           Children who have experienced this type of attachment may:
 
-          - continuously seek approval
-          - have a fear of being abandoned
-          - cling to the parent or carer
-          - lack the security to settle into learning
+            - continuously seek approval
```

Comment:
list indentation might not be advisable but would require rendering to confirm

Reply:
ah right for some reason i thought we always did it like this. Have looked in review app and it has come out fine though so think it's ok

Reply:
Nice one, markdown eventually treats indented stuff differently and I wasn't sure what Govpseak would do. 
https://www.markdownguide.org/basic-syntax#code-blocks-1